pytest_mypy_plugins/item.py

```diff
@@ -224,8 +224,10 @@ def typecheck_in_same_process(
             # add current directory to path
             sys.path.insert(0, str(execution_path))
 
-            with capture_std_streams() as (stdout, stderr):
-                return_code = run_mypy_typechecking(mypy_cmd_options)
+            stdout = io.StringIO()
```

Comment:
I am a bit concerned that these two are never closed.

Reply:
There does not appear the be any resource leak associated with unclosed StringIO objects AFAIK. It's just a buffer that gets garbage collected when the reference goes out of the scope. Am I missing something?

Reply:
Except that we don't know what exactly does `mypy.build.build()` does with the reference though. Would closing the buffers after `run_mypy_typechecking` is run make you more comfortable?

Reply:
Yes :)

Reply:
Done!